bundles/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/internal/discovery/LutronMcastBridgeDiscoveryService.java

```diff
@@ -188,15 +189,22 @@ private void createBridge(DatagramPacket packet) {
                 properties.put(SERIAL_NUMBER, serialNumber);
 
                 if (PRODFAM_RA2.equals(productFamily)) {
-                    properties.put("productFamily", "RadioRA 2");
+                    properties.put(PROPERTY_PRODFAM, "RadioRA 2");
                 } else if (PRODFAM_HWQS.equals(productFamily)) {
-                    properties.put("productFamily", "HomeWorks QS");
+                    properties.put(PROPERTY_PRODFAM, "HomeWorks QS");
                 } else {
-                    properties.put("productFamily", productFamily);
+                    properties.put(PROPERTY_PRODFAM, productFamily);
                 }
 
-                properties.put("productType", productType);
-                properties.put("version", codeVersion);
+                if (StringUtils.isNotBlank(productType)) {
```

Comment:
We plan to get rid of `apache.commons` dependency in OH3. Are you able to replace the usage of `StringUtils` in this class?

Reply:
Sure. Done.